docs/installing-solidity.rst

```diff
@@ -278,13 +296,17 @@ Building Solidity is quite similar on Linux, macOS and other Unices:
     cd build
     cmake .. && make
 
-or even easier:
+or even easier on Linux and macOS, you can run:
 
 .. code-block:: bash
 
     #note: this will install binaries solc and soltest at usr/local/bin
     ./scripts/build.sh
 
+.. note::
+
+    With BSD-based distributions you may also need to run ``ln -s /<path>/solidity/build/solc/solc /usr/local/bin/solc`` to add the binary to your path.
```

Comment:
I'm not sure that we need this note - in fact to me this seems like a weird thing to say - either the install is broken on bsd - then this should be noted here instead - or it works, then this is redundant/wrong.

Reply:
A symbolic link is not a good idea for this in any case anyways :-).

Reply:
@ekpyron This came from the original contributor, as noted above, I couldn't get the build to work anyway, so I think I will add a disclaimer that this is untested.

Reply:
@ChrisChinchilla I understand that - but I still think we should change ``ln -s /<path>/solidity/build/solc/solc /usr/local/bin/solc`` to ``cp /<path>/solidity/build/solc/solc /usr/local/bin/solc`` - then it should be fine.

Reply:
@ekpyron Take a look at the latest commit, removed that line completely.